.github/workflows/master_approve.yml

```diff
@@ -0,0 +1,34 @@
+---
+name: Master CI Approve Workflows
+on:
+  workflow_call:
+
+jobs:
+  
+  file-check:
+    uses: TessellDevelopment/tessell-ci/.github/workflows/master_file_check.yml@TDEVOPS-827
+    secrets: inherit
+  
+  read-codeowners:
+    needs: file-check
+    uses: TessellDevelopment/tessell-ci/.github/workflows/master_read_codeowners.yml@TDEVOPS-827
+    secrets: inherit
+    with:
+      changed-files: ${{needs.file-check.outputs.changed-files}}
+
+  post-message: 
+    if: ${{ github.event.action == 'opened' || github.event.action == 'synchronize' }}
+    needs: read-codeowners
+    uses: TessellDevelopment/tessell-ci/.github/workflows/master_post_message.yml@TDEVOPS-827
+    secrets: inherit
+    with:
+      required-teams: ${{needs.read-codeowners.outputs.required-teams}}
+
+  check-approval:
+    if: github.event.review.state == 'approved'
```

Comment:
Say, there is a team X with 2 or more members in it. And one member approves while another rejects the PR. 

What will be the overall status on the pending approval list or mergeability of the PR. 

Expected is that the PR should not be allowed to merge.